tools/autograd/derivatives.yaml

```diff
@@ -2963,3 +2963,56 @@
 - name: _reshape_copy(Tensor self, SymInt[] size) -> Tensor
   self: grad.reshape_symint(self.sym_sizes())
   result: auto_linear
+
+# note(crcrpar): `torchgen/api/autograd` logic would unwantedly replace substrings of `self` and `other` of function names.
```

Comment:
Are all the notes here things we want to fix before merging?

Reply:
I don't think these are high priority or easy to fix.

this first note: I didn't feel like changing the reference function name
the third (maximum and minimum): would you want to add overloads to the native functions?

Reply:
These two sound good to leave as is.
What about the second? weird numerical errors sound bad?

Reply:
The numerical errors looked to be caused by some naitve mistakes in torchgen changes, e.g. the grads of `_foreach_sub` seemed to be wrongly negated. I however didn't come up with a fix after comparing the generated code of `_foreach_sub` and `sub` in torch/csrc/autograd/generated/Function.cpp`